dcat/index.html

```diff
@@ -3829,6 +3829,100 @@ <h4>Property: checksum value</h4>
 
 </section>
 
+<!-- Start of Draft section on inverse properties -->
+
+<section id="inverse-properties">
+
+<h2>Use of inverse properties</h2>
+
+<p>The properties described in <a href="#vocabulary-specification"></a> do not intentionally include inverses, with the purpose of ensuring interoperability also in systems not making use of OWL reasoning.</p>
```

Comment:
Contrary to what is written before the table listing the inverse, `dcterms:isReplacedBy `and `dcat:hasVersion` are included in section 6. 
Perhaps adjust the first sentence in the paragraph,  
```suggestion
<p>Not all the properties described in <a href="#vocabulary-specification"></a>  include inverses, this with the purpose of ensuring interoperability also in systems not making use of OWL reasoning.</p>
```
Otherwise, we might consider dropping the inverse from section 6.

Reply:
I think we should consider dropping those properties from section 6, and keeping them only here.

Reply:
The recent revisions acknowledge this point (that we do have some inverses in section 6) explicitly so I think its more important to merge this and have the broader group discussion.  So I've marked this as resolved and we can address when we have consensus.

Reply:
Thanks, @davebrowning